src/gcp_scanner/crawl.py

```diff
@@ -661,8 +657,8 @@ def get_bigtable_instances(project_id: str,
         parent=f"projects/{project_id}")
     while request is not None:
       response = request.execute()
-      for instance in response.get("instances", []):
-        bigtable_instances_list.append(instance)
+      bigtable_instances_list = [instance 
+                                 for instance in response.get("instances", [])]
```

Comment:
This is redundant. Replace with `bigtable_instances_list = response.get("instances", [])`

Reply:
I have made the requested changes in the latest PR. Check and let me know if any changes required.